recipes/aws-sdk-cpp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "aws-sdk-cpp" %}
+{% set version = "1.4.19" %}
+{% set sha256 = "1c31651ef79b77a1fe94b0533c1ccd7b3f8eb037a46b0c723143fe5da0bf7a1e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/aws/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+ number: 0
+ skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.2
+    - curl
+    - openssl
```

Comment:
Can you please add the pinning for these two pinned here and below?

```
- curl >=7.44.0,<8
- openssl 1.0.*
```

PS: note that we are in the process to change how we deal with pinned dependencies,
but we move to conda-build 3 we still need to add the pins here.